Scripts/ReadPDFFileV2/ReadPDFFileV2.py

```diff
@@ -66,10 +66,6 @@ def run_shell_command(command, *args):
                              f' Error: {completed_process.stderr}')
     elif completed_process.stderr:
         # raise only if stderr contains non warning messages
-        lines = completed_process.stderr.splitlines()
-        for l in lines:
-            if 'warning:' not in l.lower():
-                raise ShellException(f'{completed_process.stderr}Error code: {completed_process.returncode}')
```

Comment:
better to filter the specific error, if it works fine with it, and not to delete the whole error handling